src/js/embeds.js

```diff
@@ -294,8 +294,7 @@
         }
         var pastedUrl = e.originalEvent.clipboardData.getData('text');
 
-        if (regexHelper.vimeo_parser(pastedUrl) ||
-            regexHelper.youtube_parser(pastedUrl)) {
+        if (regexHelper.urlParser(pastedUrl)) {
```

Comment:
Ok I can agree to this. I'll remove regexHelper and check for a simple http regular expression.
Sorry it took so long, I was sick for some time and also went on holiday :)